Sources/Workspace/Workspace.swift

```diff
@@ -513,6 +529,10 @@ public class Workspace {
             RepositoryPackageConstraint(
                 container: dependency.repository, requirement: requirement))
 
+        // Add unversioned constraints for edited packages.
+        let currentManifests = loadDependencyManifests(rootManifests: rootManifests, engine: engine)
```

Comment:
lets move this to be just after loadRootManifests, so we return early in case engine contains errors